packages/service-library/src/servicelib/db_async_engine.py

```diff
@@ -17,6 +18,12 @@
 
 @retry(**PostgresRetryPolicyUponInitialization(_logger).kwargs)
 async def connect_to_db(app: FastAPI, settings: PostgresSettings) -> None:
```

Comment:
confusion here. why is this not in the fastapi subfolder?
this is a duplicate?

Reply:
There is a common, then two variants: one for aiotthp and another for fastapi